api/src/main/java/org/openmrs/module/dhisconnector/api/impl/DHISConnectorServiceImpl.java

```diff
@@ -1479,6 +1499,21 @@ private int getHalfYear(Calendar date) {
 	private int getHalfYearApril(Calendar date) {
 		return date.get(Calendar.MONTH) < Calendar.OCTOBER && date.get(Calendar.MONTH) >= Calendar.APRIL ? 1 : 2;
 	}
+
+	/**
+	 * @return the quarter number
+	 */
+	private int getQuarterNumber(Calendar date) {
+		if (date.get(Calendar.MONTH) < Calendar.APRIL) {
+			return 4;
```

Comment:
This should be change to 1

Reply:
oh yeah right, Thank you for pointing it out :) 
Will do

Reply:
Resolved with the commit [9d55e16](https://github.com/openmrs/openmrs-module-dhisconnector/pull/26/commits/9d55e1621cb5bbdb355c080e8c2980fdf85fe5eb)
Please Review